src/components/data-management/ProjectCard.jsx

```diff
@@ -58,10 +58,13 @@ const ProjectCard = (props) => {
   };
 
   const updateProjectName = (newName) => {
-    dispatch(updateProject(project.uuid, { name: newName.trim() }));
-
-    // Before multiple experiment support, use project name for analysis name
-    dispatch(updateExperiment(projectExperiment, { name: newName.trim() }));
+    dispatch(updateProject(project.uuid, { name: newName.trim() })).then((e) => {
```

Comment:
If you not use .then and updateProject crashes will it still do updateExperiment too?

Reply:
actually the .then is completely unnecessary, without the if both would be triggered even when the first fails

Reply:
I realized why, the action will always succeed but it's synchronous, meaning that the if will always run after the action which is ok